pkgs/development/python-modules/measurement/default.nix

```diff
@@ -1,15 +1,26 @@
-{ lib, fetchPypi, buildPythonPackage, pbr, six, sympy }:
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k
+, sympy, pytest, pytestrunner, sphinx, setuptools_scm }:
 
 buildPythonPackage rec {
   pname = "measurement";
   version = "3.2.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "352b20f7f0e553236af7c5ed48d091a51cf26061c1a063f46b31706ff7c0d57a";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "coddingtonbear";
+    repo = "python-measurement";
+    rev = version;
+    sha256 = "1mk9qg1q4cnnipr6xa72i17qvwwhz2hd8p4vlsa9gdzrcv4vr8h9";
   };
 
-  propagatedBuildInputs = [ pbr six sympy ];
+  postPatch = ''
+    sed -i 's|use_scm_version=True|version="${version}"|' setup.py
+  '';
+
+  checkInputs = [ pytest pytestrunner ];
+  nativeBuildInputs = [ sphinx setuptools_scm ];
```

Comment:
What is sphinx used for here?

Reply:
Release 3.0 says "Update Sphinx build process"
https://github.com/coddingtonbear/python-measurement/releases

And removing it from the build inputs causes a failure, because the build script looks for it:
https://github.com/coddingtonbear/python-measurement/blob/master/setup.cfg#L29

Reply:
still part of setup_requires, I'm fine with this:

https://github.com/coddingtonbear/python-measurement/blob/43b243818e082c295d336dc2c7de23831f7c736e/setup.cfg#L27
```
setup_requires =
    setuptools_scm
    sphinx
    pytest-runner
```